### DISCUSSION TITLE
Full Node not attempting enough

### DISCUSSION THREAD

**Discussion by SackofFluor** [Chia-Network/chia-blockchain#3765] [Support]
Hey Guys,

i have a quick question and i hope somebody may be able to help me...

My full-node Farmer is sometimes not getting attempts for minutes in end...
Often it runs fine so around 1 attempt every 10sec., but then I have sometimes > 3 minutes of not a single attempt at all..
Does anybody has any idea how that can be and how I can prevent that?

Because if I am on average missing out on 10 or 20% of the attempts my chances to win anything are technically minuscule (besides the amount of plotted TiB)

EDIT:
Challenge | Plots Passed Filter | Proofs Found | Date
-- | -- | -- | --
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 0 / 230 | 0 | May 5, 8:19:22 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 1 / 230 | 0 | May 5, 8:19:15 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 0 / 230 | 0 | May 5, 8:54:15 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 1 / 230 | 0 | May 5, 8:18:13 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 1 / 230 | 0 | May 5, 8:18:06 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 1 / 230 | 0 | May 5, 8:17:23 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 1 / 230 | 0 | May 5, 8:17:10 AM
0xf55bcd7955fa1a260bc02bbf237fa2671607fa4b4eba3191c94b6493dd785c2a | 0 / 230 | 0 | May 5, 8:16:55 AM

And then, in general I have attempts missing.....

EDIT 2:
0x902eecadc80b1d1ade9e394bea4f186ca500dc7092443ade8fe2abce67ca4fa3 | 0 / 230 | 0 | May 5, 8:23:52 AM
0x902eecadc80b1d1ade9e394bea4f186ca500dc7092443ade8fe2abce67ca4fa3 | 0 / 230 | 0 | May 5, 8:23:33 AM
0x902eecadc80b1d1ade9e394bea4f186ca500dc7092443ade8fe2abce67ca4fa3 | 0 / 230 | 0 | May 5, 8:23:13 AM
0x902eecadc80b1d1ade9e394bea4f186ca500dc7092443ade8fe2abce67ca4fa**3 | 0 / 230 | 0 | May 5, 8:23:07 AM**
0x902eecadc80b1d1ade9e394bea4f186ca500dc7092443ade8fe2abce67ca4fa**3 | 0 / 230 | 0 | May 5, 8:21:53 AM**

Over 1 minute gap... wtf....

**Comment by Muffexx** [comment]
Hello @SackofFluor,

check this out:
> Plots Passed Filter: There is an algorithm that ensures your plot has 1/512 chances of being passing a filter where a plot should pass the filter about eighteen times in 24-hour period.
> _NOTES: You should see regular occurrences of your plots passing the filter, like 1 / Number of plots in your farm, instead of 0 / Number of plots in your farm._
> - [sungkim11](https://sungkim11.medium.com/farm-chia-coin-xch-and-make-money-8fda188b8689)

From my understanding everything seems to work correctly.

regards
Muffexx

**Reply by SackofFluor** [reply]
But if iam not wrong there are new challenges every 10-15 sec. So if my farmer does not get challenged for a minute your plots could have not won in the first place

**Reply by nomadengineer** [reply]
Challenges are there but your plot passes the filter less frequently! 1/230 means of your plots passed the first filter. every 0/230 is a challenge attempt.

**Comment by SackofFluor** [comment]
Wow.. it really works great......

Over 2h breaks between attempts.... sure farming will be working great...

Sorry but this is getting ridiculous....

"Pool" is not possible, yet one Pool owns now around 50% of the whole network.

For two days people get weird errors that the plots are "double-spend"... so far no reliable reply if that is a problem now or not...




<html>
<body>
<div class="MuiBox-root jss4090 Card__StyledCardTitle-g4st3-0 ktPsZi" style="box-sizing: inherit; padding: 16px; color: rgb(255, 255, 255); font-family: Roboto, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: 0.14994px; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="MuiBox-root jss4091 Flex__StyledGapBox-sc-2jumy6-0 HMfwK" style="box-sizing: inherit; display: flex; flex-wrap: wrap; align-items: center;"><div class="MuiBox-root jss4092" style="box-sizing: inherit; flex-grow: 1; margin-bottom: 0px; margin-right: 0px;"><h5 class="MuiTypography-root MuiTypography-h5" style="box-sizing: inherit; margin: 0px; font-size: 1.5rem; font-family: Roboto, Helvetica, Arial, sans-serif; font-weight: 400; line-height: 1.334; letter-spacing: 0em;"><div class="MuiBox-root jss4093 Flex__StyledGapBox-sc-2jumy6-0 brEkR" style="box-sizing: inherit; display: flex; align-items: center;"><div class="MuiBox-root jss4094" style="box-sizing: inherit; margin-bottom: 0px; margin-right: 8px;">Last Attempted Proof</div><svg class="MuiSvgIcon-root TooltipIcon__StyledHelpIcon-sc-1oqr2q2-0 pdjJL MuiSvgIcon-colorDisabled" focusable="false" viewBox="0 0 24 24" aria-hidden="true"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 17h-2v-2h2v2zm2.07-7.75l-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25z"></path></svg></div></h5></div></div></div><div class="MuiCardContent-root" style="box-sizing: inherit; padding: 16px 16px 24px; color: rgb(255, 255, 255); font-family: Roboto, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: 0.14994px; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="MuiBox-root jss4095 Flex__StyledGapBox-sc-2jumy6-0 OuzVX" style="box-sizing: inherit; display: flex; flex-direction: column;"><div class="MuiBox-root jss4096 Flex__StyledGapBox-sc-2jumy6-0 YWMuC" style="box-sizing: inherit; display: flex; flex-direction: column; margin-bottom: 0px;"><div class="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded" style="box-sizing: inherit; color: rgb(255, 255, 255); transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; background-color: rgb(66, 66, 66); border-radius: 4px; box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 1px -1px, rgba(0, 0, 0, 0.14) 0px 1px 1px 0px, rgba(0, 0, 0, 0.12) 0px 1px 3px 0px; width: 982px; overflow-x: auto; margin-bottom: 0px;">

Challenge | Plots Passed Filter | Proofs Found | Date
-- | -- | -- | --
0xd8e4bec3c402c5217fd58daeed356eaa729af2c727fba520df8a4dbf0f0d4860 | 0 / 294 | 0 | May 7, 3:11:48 PM
0xa9a468d9e1886efd6b2d0f5b106e2ef7543df2d642b2ebbea9dafc52d146c4a8 | 1 / 291 | 0 | May 7, 1:08:15 PM
0x620afec6c8ef55d012fa094f3755cf97752c4b6965d40bd8250b543df50582de | 1 / 291 | 0 | May 7, 1:19:42 PM
0xd6d8a6e8a88012fe384c838e019b08a7f69e093fcfbaf03cab8ec22dcc68a02c | 0 / 291 | 0 | May 7, 12:28:52 PM
0x8956bbf92daf766d956206bf02ae4b6e68113c30c5988193a31d4e342349d93f | 2 / 292 | 0 | May 7, 12:23:42 PM

</div></div></div></div>
</body>
</html>

**Reply by SackofFluor** [reply]
Anyone knowing what this could be ??

2021-05-07T15:49:47.891 full_node chia.rpc.rpc_server     : WARNING  Sending data failed. Exception Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 67, in _state_changed
  File "aiohttp\client_ws.py", line 150, in send_str
  File "aiohttp\http_websocket.py", line 687, in send
  File "aiohttp\http_websocket.py", line 643, in _send_frame
  File "aiohttp\http_websocket.py", line 660, in _write
ConnectionResetError: Cannot write to closing transport

**Reply by jmjava** [reply]
I am getting this also.  Chia crashes.  I see a bunch of these in debug log.  I have files in my temp directory but nothing makes it into the final directory.

**Reply by happycouak** [reply]
@SackofFluor maybe you should fill an issue about that.

**Comment by aart-farmer** [comment]
I have the same problem, did anyone find the solution?

**Reply by aart-farmer** [reply]
i did it: https://helpdeskgeek.com/help-desk/how-to-fix-rpc-server-is-unavailable-error-in-windows/

**Comment by BrandtH22** [comment]
@SackofFluor , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
